lib/chromium/preview.js

```diff
@@ -58,7 +58,19 @@ let PreviewDataLoaders = {
     });
     let name = getProperty(result, "name").value;
 
-    handle.preview = {name};
+    let {details} = yield rpc.request("Debugger.getFunctionDetails", {
+      functionId: handle.objectId
+    });
+    if (!details) {
+      handle.preview = {name};
+      return;
+    }
+
+    let functionName = details.functionName;
+    let inferredName = details.inferredName;
+    let displayName = details.displayName;
+
+    handle.preview = { name, functionName, inferredName, displayName };
```

Comment:
nit: to be consistent with the rest of the file, no whitespace after { and before } should be added.